docs/en/operation/Basic-Logging.md

```diff
@@ -1,43 +1,68 @@
 ---
 layout: global
-title: Server Logging
-nickname: Server Logging
+title: Basic Logging
+nickname: Basic Logging
 group: Operations
-priority: 12
+priority: 11
 ---
 
 * Table of Contents
 {:toc}
 
 ## Introduction
 
-This page summarizes Alluxio's logging system for servers, such as the master and worker
-processes. We include tips for modifying Alluxio's log4j properties file to best suit
-deployment needs. If you are a user of Alluxio looking for logging information about clients
-that are utilizing the Alluxio API, we recommend looking at the
-[client logging documentation]({{ '/en/operation/Client-Logging.html' | relativize_url }})
-
-Alluxio's logging behavior can be fully configured through the `log4j.properties` file found in the
-`conf` folder.
+This page describes the basic logging function provided by Alluxio server processes (e.g., masters, workers and etc)
+and application processes utilizing Alluxio clients (e.g., Spark or MapReduce jobs running on Alluxio).
+Note that, there is an advanced feature that streams server-side and client-side logs to separate Alluxio logservers
+(see [remote logging]({{ '/en/operation/Remote-Logging.html' | relativize_url }}) for more details).
 
 ## Log Location
 
-### Default Location
+Alluxio logging function is based on [`log4j`](https://logging.apache.org/log4j/) library,
+configured through the `${ALLUXIO_HOME}/conf/log4j.properties`.
+Logs are written to different files at different locations by Alluxio server processes and  application processes:
```

Comment:
over explanation of application process logs. can simplify to
```
Log files for each individual Alluxio server process can be found under `${ALLUXIO_HOME}/logs/`.
Log files for the Alluxio client utilized by different applications are located with their respective application logs.
```

i might even suggest removing the application-side logs section because it's so short. could continue from the above lines and go directly into:
```
Below are links to the documentation of particular compute frameworks with information
on where logs may be found and how to configure them.

- [Apache Hadoop]({{ '/en/compute/Hadoop-MapReduce.html' | relativize_url }}#logging-configuration)
- [Apache HBase]({{ '/en/compute/HBase.html' | relativize_url }}#logging-configuration)
- [Apache Hive]({{ '/en/compute/Hive.html' | relativize_url }}#logging-configuration)
- [Apache Spark]({{ '/en/compute/Spark.html' | relativize_url }}#logging-configuration)
```
then proceed to describe server side logs in more detail

Reply:
refactored this section